context/app/templates/pages/base.html

```diff
@@ -9,6 +9,7 @@
     <!-- <link rel="stylesheet" href="{{ url_for('static', filename='todo-styles.css') }}"> -->
     <link rel="stylesheet" href="{{ url_for('static', filename='table.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='sticky-footer.css') }}">
+    <link rel="shortcut icon" image="image/ico" href="../../static/public/assets/favicon.ico">
```

Comment:
- Incuding "shortcut" is [no longer a good practice](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types). (See the "icon" row.)
- I don't think the `image` is necessary... and if you do want a MIME type, the attribute should be `type`.
- You're trying to use a relative path from the source code, but what would `../../` mean if you were on `/`?

Reply:
I believe you should use `url_for()` to access files in static/public similar to the stylesheets imported above the favicon